routers/repo/branch.go

```diff
@@ -203,10 +204,16 @@ func loadBranches(ctx *context.Context) []*Branch {
 			}
 		}
 
+		isMerged := true
+		if (divergence.Ahead != 0) || (divergence.Behind == 0) || (ctx.Repo.Repository.DefaultBranch == branchName) {
```

Comment:
How about to change this to

```
isMerged := false
if (divergence.Ahead == 0 && divergence.Behind > 0 && ctx.Repo.Repository.DefaultBranch != branchName)  {
     isMerged = true
}
```

Reply:
well the result will be the same
 + it if false by default
 -  it has to check each condition to set it to true (speed)